charts/keycloak-operator/Chart.yaml

```diff
@@ -16,7 +16,7 @@ maintainers:
 annotations:
   artifacthub.io/changes: |
     - kind: changed
-      description: "chore: update keycloak from v26.0.1 to v26.0.6"
+      description: "chore: update keycloak to v26.0.7"
       links:
         - name: "Keycloak 26.0.2"
```

Comment:
Also, the old links need removing... artifacthub aggregates the releasenotes over time